internal/pkg/metadb/timestamps/timestamps.go

```diff
@@ -23,6 +23,11 @@ import (
 
 const (
 	signaturePropertyName = "Signature"
+
+	// Precision is the effective precision of timestamps.
+	// Currently it's 1 microsecond as it's what Datastore supports.
+	// Datastore: 1 microsecond: https://cloud.google.com/datastore/docs/concepts/entities#date_and_time
+	Precision = time.Millisecond
```

Comment:
I am not sure to understand the comments here. 1 micro seconds is 1e-6... 1 milli second is 1e-3.